src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -53,14 +51,10 @@ class PlannerPlanDetailsGetCommand extends GraphItemsListCommand<any> {
           this.groupId = groupId;
           return this.getPlanId(args);
         })
-        .then((planId: string): Promise<any> => {
+        .then((planId: string): any => {
```

Comment:
Let's not use `any`. Since we know the type of data we'll get, let's use specific types to avoid runtime errors.

Reply:
done